mobile/src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -234,42 +234,40 @@ public void getFeeds(@NonNull Callback<List<Feed>> callback) {
     }
 
     public void getOpreations(@NonNull Callback<List<Operation>> callback, Resource resource) {
-        URI uri = Uris.getUri(String.format(BackendPipes.Paths.OPERATIONS,
-                CanonicalPath.getByString(resource.getPath()).getFeed(),
-                Uris.getEncodedParameter(resource.getType().getId())));
+        URI uri = Uris.getUri(CanonicalPath.getByString(resource.getType().getPath())
+                .fix(BackendPipes.Paths.OPERATIONS));
 
         readPipe(BackendPipes.Names.OPERATIONS, uri, callback);
     }
 
     public void getResourcesFromFeed(@NonNull Callback<List<Resource>> callback, Feed feed) {
-        URI uri = Uris.getUri(String.format(BackendPipes.Paths.FEED_RESOURCES,
-                Uris.getEncodedParameter(feed.getId())));
+        URI uri = Uris.getUri(CanonicalPath.getByString(feed.getPath()).fix(BackendPipes.Paths.FEED_RESOURCES));
 
         readPipe(BackendPipes.Names.FEED_RESOURCES, uri, callback);
     }
 
 
     public void getRecResourcesFromFeed(@NonNull Callback<List<Resource>> callback, Resource resource) {
 
-            URI uri = Uris.getUri(String.format(BackendPipes.Paths.FEED_REC_RESOURCES,resource.getFeed(),
```

Comment:
All resources are no longer used in code anywhere. So I think it would be good to remove them. Adding commit for renaming those metrics and removing all resource collection part